RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/code_models/RankEvaluater.java

```diff
@@ -0,0 +1,79 @@
+package org.jetbrains.research.extractMethodExperiments.code_models;
+
+import org.apache.commons.lang3.StringUtils;
+
+import static org.jetbrains.research.extractMethodExperiments.code_models.Fragment.getNestingArea;
+import static org.jetbrains.research.extractMethodExperiments.code_models.Fragment.getNestingDepth;
+
+public class RankEvaluater {
+    private String candidate;
+    private String remainder;
+    private String method;
+    private double score;
+    private int methodDepth;
+    private int methodArea;
+    private double lengthScoreSensitivity;
+    private double maxLengthScore;
+
+    public RankEvaluater(Fragment.SubFragment sf, String initialMethod, int methodArea, int methodDepth) {
+        this.candidate = sf.getBody();
+        this.method = initialMethod;
+        this.remainder = sf.getRemainder();
+        this.methodDepth = methodDepth;
+        this.methodArea = methodArea;
+
+        this.lengthScoreSensitivity = 0.1;
```

Comment:
you can define the field's value in its declaration

`private final double lengthScoreSensitivity = 0.1;`
`private final double maxLengthScore = 3;`